### ISSUE TITLE
Order of people in the list should be deterministic

### ISSUE THREAD

**Issue by sahildua2305** [lenadroid/awesome-oss-mentors#6]
I'd like to propose to have the list of awesome mentors in alphabetical order of their first names.

**Comment by 1ForA11** [comment]
I also think that sorting based on the skill level is also important, maybe Alphabitcally in the same skill scope?

**Comment by sahildua2305** [comment]
> maybe Alphabitcally in the same skill scope?

Yes, that's what I thought of initially. How do we define a skill scope? It's very subjective.

**Comment by 1ForA11** [comment]
Not what I meant, I just want to group people with the same skill level together no matter how far they are in the table. For example, when I am looking for Java, I want to see all the Java people and not have to cherry pick them from the entire table, I.e, people with the same skill level should be close in proximity, and if possible, sorted by alphabetical order.

**Comment by lenadroid** [comment]
This might be challenging to do, because a lot of mentors have _many_ focus areas...

Do you have any example in mind of how it should look like? If you take the current list as input.

**Comment by andy-shev** [comment]
Just oblige to split areas to main + aux ones. Sort by main area and by name inside each group.

**Comment by andy-shev** [comment]
People with * there would not be in the common list -- separate group at the end, or better to insist them to emphasize main area.

**Comment by lenadroid** [comment]
Thanks @andy-shev! That would make sense for new mentors. Not sure how to handle existing ones, because I personally don't know what is the main area, and it would be a little cumbersome to contact all of them to fix the description. Sorry I didn't think it through from the very beginning, and answers from different people vary a lot! I'll think about how to handle this.

**Comment by andy-shev** [comment]
Lemme do a PR to start a Linux kernel group then.

**UPDATE**. Send PR #35

**Comment by 1ForA11** [comment]
@lenadroid sorry for the late reply. I like the idea of groups as a middle ground solution to avoid the complexity of having to sort out people with many skill sets. For example, the containers group ( Azure, Kubernetes, Docker, ...). A person with skills spanning multiple groups can be in more than one group at a time.
